platforms/joystick/joystick_nintendo_joycon.go

```diff
@@ -0,0 +1,115 @@
+package joystick
+
+var joyconPairConfig = joystickConfig{
+	Name: "Nintendo Switch Joycon Controller Pair",
+	GUID: "5555",
```

Comment:
Not sure where to get this, happy to investigate more with some pointers.  In other examples I looked at it seemed random values were being used and I didn't see GUID mentioned in the rest of the codebase anyways.